packages/xrpl/src/Wallet/rfc1751.ts

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers -- Doing many bitwise operations which need specific numbers */
+/* eslint-disable no-bitwise -- Bitwise operators are required for this encoding/decoding */
+/* eslint-disable id-length -- Bitwise math uses shorthand terms */
+/*
+ *rfc1751.ts : Converts between 128-bit strings and a human-readable
+ *sequence of words, as defined in RFC1751: "A Convention for
+ *Human-Readable 128-bit Keys", by Daniel L. McDonald.
+ *Ported from rfc1751.py / Python Cryptography Toolkit (public domain).
+ *Copied from https://github.com/bip32/bip32.github.io/blob/master/js/rfc1751.js which
+ *is part of the public domain.
+ */
+
+import rfc1751Words from './rfc1751Words.json'
+
+const rfc1751WordList: string[] = rfc1751Words
+
+// prettier-ignore - It's more readable to have this on two lines than one per line.
+const binary = [
```

Comment:
Looks like the linter made it one per line anyways

Reply:
nit: this should be
```suggestion
const _BINARY = [
```

Reply:
Adding the comment made the `// prettier-ignore` stop working, so I removed the comment